packages/mobile/src/backup/BackupIntroduction.tsx

```diff
@@ -58,170 +52,104 @@ class BackupIntroduction extends React.Component<Props> {
-            </>
-          )}
-        </ScrollView>
-        <View>
-          {!backupCompleted && (
-            <>
-              <Button
-                onPress={this.onPressBackup}
-                text={t('getYourKey')}
-                standard={false}
-                type={BtnTypes.PRIMARY}
-              />
-              {backupTooLate && !backupDelayedTime && (
-                <Button
-                  onPress={this.onPressDelay}
-                  text={t('delayBackup')}
-                  standard={false}
-                  type={BtnTypes.SECONDARY}
-                />
-              )}
-            </>
-          )}
-
-          {backupCompleted && !socialBackupCompleted && (
-            <>
-              <Button
-                onPress={this.onPressSetupSocialBackup}
-                text={t('setUpSocialBackup')}
-                standard={false}
-                type={BtnTypes.PRIMARY}
-              />
-              <Button
-                onPress={this.onPressViewBackupKey}
-                text={t('viewBackupKey')}
-                standard={false}
-                type={BtnTypes.SECONDARY}
-              />
-            </>
-          )}
-
-          {backupCompleted && socialBackupCompleted && (
-            <>
-              <Button
-                onPress={this.onPressBackup}
-                text={t('viewBackupKey')}
-                standard={false}
-                type={BtnTypes.SECONDARY}
-              />
-              <Button
-                onPress={this.onPressViewSocialBackup}
-                text={t('viewSafeguards')}
-                standard={false}
-                type={BtnTypes.SECONDARY}
-              />
-            </>
-          )}
-        </View>
+        {backupCompleted ? (
+          <AccountKeyPostSetup />
+        ) : (
+          <AccountKeyIntro onPrimaryPress={this.onPressBackup} />
+        )}
       </SafeAreaView>
     )
   }
 }
 
+interface AccountKeyStartProps {
+  onPrimaryPress: () => void
+}
+
+function AccountKeyIntro({ onPrimaryPress }: AccountKeyStartProps) {
+  const { t } = useTranslation(Namespaces.backupKeyFlow6)
+  return (
+    <ScrollView contentContainerStyle={styles.introContainer}>
+      <Logo height={32} />
+      <Text style={styles.h1}>{t('introTitle')}</Text>
+      <Text style={styles.body}>{t('introBody')}</Text>
+      <Button text={t('introPrimaryAction')} onPress={onPrimaryPress} />
+    </ScrollView>
+  )
+}
+
+function AccountKeyPostSetup() {
+  const accountKey = useAccountKey()
+
+  const { t } = useTranslation(Namespaces.backupKeyFlow6)
+
+  return (
+    <ScrollView contentContainerStyle={styles.postSetupContainer}>
+      <View>
+        <Text style={fontStyles.h2}>{t('introTitle')}</Text>
+        <View style={styles.keyArea}>
+          <Text style={fontStyles.large}>{accountKey}</Text>
+        </View>
+        <Text style={styles.postSetupBody}>{t('postSetupBody')}</Text>
+      </View>
+      <View style={styles.postSetupCTA}>
```

Comment:
ditto about moving button out of `ScrollView`

Reply:
confirmed with Cory it should not be taken out